ika/util.py

```diff
@@ -49,14 +49,17 @@ async def _write_and_read(self, command):
                 response = await self._handle_communication(command)
                 if response is None:
                     return None
-                elif 'IN_NAME' in command or 'TYPE' in command:
+                elif 'IN_VERSION' in command or 'IN_NAME' in command or 'TYPE' in command:
                     return response
                 elif 'IN_PV_4' in response:
                     raise ConnectionError(
                         'Hotplate configured to communciate with a Eurostar overhead stirrer. '
                         'This must be turned off in the hotplate settings.'
                     )
+                elif command in response:  # vacuum replies to queries by echoing the query
```

Comment:
The vacuum replies differently than other devices!  However, I think this change is safe.

Reply:
Tested on Adams; it appears fine.